deps/wazuh_testing/wazuh_testing/api.py

```diff
@@ -20,6 +20,7 @@
 API_USER = 'wazuh'
 API_PASS = 'wazuh'
 API_LOGIN_ENDPOINT = '/security/user/authenticate'
+API_GLOBAL_TIMEOUT = 5
```

Comment:
In order to not affect the rest of the API test, change this to 20
```suggestion
API_GLOBAL_TIMEOUT = 20
```